src/main/java/de/tum/in/www1/artemis/web/rest/LongFeedbackTextResource.java

```diff
@@ -33,17 +34,17 @@ public LongFeedbackTextResource(LongFeedbackTextRepository longFeedbackTextRepos
      *
      * @param resultId   The result the feedback belongs to.
      * @param feedbackId The feedback for which the long feedback should be fetched.
-     * @return The long feedback belonging to the feedback with id {@code feedbackId}.
+     * @return The long feedback text belonging to the feedback with id {@code feedbackId}.
      */
     @GetMapping("results/{resultId}/feedbacks/{feedbackId}/long-feedback")
     @EnforceAtLeastStudent
-    public ResponseEntity<LongFeedbackText> getLongFeedback(@PathVariable Long resultId, @PathVariable Long feedbackId) {
+    public ResponseEntity<String> getLongFeedback(@PathVariable Long resultId, @PathVariable Long feedbackId) {
```

Comment:
It's not directly needed, but I discussed that change with @b-fein and we decided to change that directly since I'm already working on that place.

The client never accessed other properties than the text and the missing `@JsonIgnore` was due to a merging issue, it should have been there from the beginning.

Reply:
Ok. After merge, I'll adapt [artemis4j](https://github.com/kit-sdq/artemis4j/blob/eca75a7b2b758e1ca3c64dbe618d67a93834a304/src/main/java/edu/kit/kastel/sdq/artemis4j/client/AssessmentArtemisClient.java#L165-L169)